src/ConnectedMode.UnitTests/QualityProfiles/QualityProfileDownloaderTests.cs

```diff
@@ -61,20 +61,19 @@ public async Task UpdateAsync_NothingToUpdate_ReturnsFalse()
                 boundSonarQubeProject,
                 Array.Empty<Language>());
             var logger = new TestLogger();
-            var solutionBindingOperationMock = new Mock<ISolutionBindingOperation>();
+
+            var bindingConfigProvider = new Mock<IBindingConfigProvider>();
 
             var testSubject = CreateTestSubject(outOfDateQualityProfileFinderMock.Object,
+                bindingConfigProvider.Object,
                 logger: logger,
-                solutionBindingOperation: solutionBindingOperationMock.Object,
                 languagesToBind: Language.KnownLanguages.ToArray());
 
             var result = await testSubject.UpdateAsync(boundSonarQubeProject, null, CancellationToken.None);
 
             result.Should().BeFalse();
-            solutionBindingOperationMock.Verify(
-                x =>
-                    x.SaveRuleConfiguration(It.IsAny<IEnumerable<IBindingConfig>>(), It.IsAny<CancellationToken>()),
-                Times.Never);
+            bindingConfigProvider.Invocations.Should().BeEmpty();
```

Comment:
The `ISolutionBindingConfiguration` class doesn't exist anymore, so instead we're checking the config provider is never called as an indicator that the method exited.